LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2016 keyring Developers
```

Comment:
one tweak though the license file should probably replace "keyring" with "tesseract"?

Reply:
and year to 2019

Reply:
Thanks for the catch :)

Reply:
I don't know if you can test it before you merge this, but I think if you name the file as something like `LICENSE.md` GitHub picks it up and shows in the project navigation on the main page.